app/src/main/java/com/kms/cura/utils/DataUtils.java

```diff
@@ -81,5 +82,15 @@ private static Date gettheCurrent() {
         return date;
     }
 
+    public static List<AppointmentEntity> getApptByDate(List<AppointmentEntity> appts, Date date){
+        List<AppointmentEntity> newList = new ArrayList<>();
+        for (AppointmentEntity entity: appts){
+            if (date.toString().equals(entity.getApptDay().toString())){
```

Comment:
@hieumai the date from the database and the date from app have different milliseconds so we can not compare